bundles/org.eclipse.e4.ui.workbench.addons.swt/src/org/eclipse/e4/ui/workbench/addons/minmax/TrimStack.java

```diff
@@ -334,7 +334,9 @@ private void fixToolItemSelection() {
 			for (ToolItem item : trimStackTB.getItems()) {
 				item.setSelection(false);
 			}
-		} else if (isEditorStack() || minimizedElement instanceof MPlaceholder) {
+		} else if ((isEditorStack() || minimizedElement instanceof MPlaceholder)
+				&& trimStackTB.getItems().length >= 1) {
```

Comment:
1) I would prefer something like `trimStackTB.getItemCount()`
That would not require to create an array
2) Why is this `>= 1` and not `> 1` ? If I understand it right, we want select **second** item, whatever it is, so it should be `trimStackTB.getItemCount() > 1`

Reply:
I agree.
Fixed.